test.js

```diff
@@ -670,6 +675,108 @@ function buildTests (test, sync) {
       }, 0)
     })
   })
+
+  test('mode', (t) => {
```

Comment:
Please skip when using windows.

Reply:
Oh... my "fix" for windows didn't work 😞 
Yes I'll skip!

Reply:
None of this is really supported on Windows. It's the unix permission system. Use the `skip` option of tap's `test` function.

Reply:
yup makes sense! :)